metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -169,6 +171,26 @@ object UserConfiguration {
            |
            |""".stripMargin
       ),
+      UserConfigurationOption(
+        "type-annotations-enabled",
+        "false",
+        "false",
+        "Should display type annotations for infered types",
+        """|When this option is enabled, for each method that can have infered types they are 
+           |displayed either as additional decorations if they are supported for the editor or 
+           |as info diagnostics otherwise.
```

Comment:
```suggestion
        """|When this option is enabled, each method that can have inferred types has them 
           |displayed either as additional decorations if they are supported by the editor or 
           |as info diagnostics.
```

Reply:
Fixed and updated with info about hover.